packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -144,6 +222,7 @@ export class AndroidEmulatorDevice extends DeviceBase {
         "-no-boot-anim",
         "-grpc-use-token",
         "-no-snapshot-save",
+        "-writable-system",
```

Comment:
what is this option? why are we adding it here? It should be explained in PR description